setuptools/py27compat.py

```diff
@@ -4,7 +4,7 @@
 
 import platform
 
-import six
+from setuptools.extern import six
```

Comment:
Why using six???

In package six the Python version is determined by calling sys.

PY2 = sys.version_info[0] == 2
See: https://bitbucket.org/gutworth/six/src/92e1c746e1abfbdb94705b821bc93766083efc54/six.py?at=default&fileviewer=file-view-default

Best solution is to use "sys" to check if it is python 2 version.

If it must be version 2.7 check for major and minor version!